google-cloud-bigquery/pom.xml

```diff
@@ -81,6 +81,10 @@
       <groupId>org.threeten</groupId>
       <artifactId>threetenbp</artifactId>
     </dependency>
+    <dependency>
+      <groupId>org.json</groupId>
```

Comment:
We should check with our legal team if we can use this package as a dependency.

There is a line in their license that makes this not the standard MIT open source license.

https://github.com/stleary/JSON-java/blob/f1b0210b8acdbc5f5316bc5507e2f3630506a9a3/LICENSE#L14

https://github.com/googleapis/google-http-java-client recommends Gson, and it should already be in our transitive dependencies.

Reply:
Thanks, removing this dependency and using `gson` instead.